test/config/test-session.data.xml

```diff
@@ -115,9 +115,11 @@
 
 
 <obj class="ConnectionService" id="local-connection-server">
- <attr name="application_name" type="string" val="gunicorn"/>
+ <attr name="application_name" type="string" val="echo"/>
```

Comment:
Was this change just for debug, surely it should be `gunicorn` rather than `echo`?